Makefile

```diff
@@ -7,14 +7,18 @@ ifneq ($(HOST),dom0)
 endif
 
 ## Builds and provisions all VMs required for testing workstation
-all: assert-dom0 validate clean update-fedora-templates			\
+all: assert-dom0 validate clean qubes-rpc update-fedora-templates			\
 	update-whonix-templates prep-whonix prep-dom0 sd-workstation-template \
 	sd-whonix sd-svs sd-gpg	\
 	sd-journalist sd-svs-disp
 
 clone: assert-dom0 ## Pulls the latest repo from work VM to dom0
 	@./scripts/clone-to-dom0
 
+qubes-rpc: prep-salt ## Places default deny qubes-rpc policies for sd-svs and sd-gpg
+	sudo qubesctl top.enable sd-dom0-qvm-rpc
+	sudo qubesctl --targets sd-dom0-qvm-rpc state.highstate
```

Comment:
Change

`sudo qubesctl --targets sd-dom0-qvm-rpc state.highstate`

to

`sudo qubesctl --show-output --targets sd-dom0-qvm-rpc state.highstate`

to match changes in #190.